### PR TITLE
Implement Soribada Connector

### PR DESCRIPTION
**Pull request by NamJinWoo**

### CODE REVIEW

**Comment by NamJinWoo** [comment]
This soribada site is a music webapp in South Korea. This is almost the first music web site in our country. Please take a look and leave me a comment. This would help your site have more datas.

**Comment by JeonHyungJin** [comment]
Maybe your code missed " , " in core/connectors.js file.
:)

**Comment by NamJinWoo** [comment]
core/connector part has been modified!

File: src/connectors/soribada.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+
```
Comment (inverse) [inline src/connectors/soribada.js]: Spacing seems a bit off in this file.

File: src/core/connectors.js [hunk 2]
```diff
@@ -967,11 +967,14 @@ define(function() {
 	},
 	{
 		label: 'Naver',
-		matches: ['*://playerui.music.naver.com*'],
+		matches: ['*://playerui.music.naver.com/*'],
```
Comment (inverse) [inline src/core/connectors.js]: Looks like I missed that one :/

File: src/connectors/soribada.js (hunk 1 above)
Comment (NamJinWoo) [reply src/connectors/soribada.js]: okay, I've done it! Sorry about it!

File: src/core/connectors.js (hunk 2 above)
Comment (NamJinWoo) [reply src/core/connectors.js]: Never Mind, I just looked around and it was a simple error!

**Comment by NamJinWoo** [comment]
I've  done all the changes requested!

File: src/connectors/soribada.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Connector.artistSelector = '.info .pado_tit_wrap span:nth-child(2)';
+
+Connector.trackSelector = '.pado_tit_wrap strong:first';
+
+Connector.currentTimeSelector = '#progress .now';
+
+Connector.durationSelector = '#progress .total';
+
+Connector.isPlaying = () => {
+
```
Comment (inverse) [inline src/connectors/soribada.js]: Could we also remove this extra spacing too?

File: src/connectors/soribada.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	return $('#controller .pause').length !== 0;
+
+};
+
+Connector.getTrackArt = () => {
+
```
Comment (inverse) [inline src/connectors/soribada.js]: And here

**Comment by inverse** [comment]
After the final bits we should be good :+1:

**Comment by NamJinWoo** [comment]
I fixed the extra spacing. Please check ! It is so nice to have a feedback with you. I'll try my best to solve many issues and contribute to this project. Let's keep in touch!

**Pull request merged**